bin/manage.js

```diff
@@ -82,7 +82,8 @@ function createClient() {
 program
   .version(pkgData.version || 'unknown')
   .description('Perform various management tasks for the Taskcluster AWS Provisioner')
-  .option('-u, --url [url]', 'URL for the API to work against', 'https://taskcluster-aws-provisioner2.herokuapp.com/v1');
+  .option('-u, --url [url]', 'URL for the API to work against', 'http://localhost:5557/v1');
+  //.option('-u, --url [url]', 'URL for the API to work against', 'https://taskcluster-aws-provisioner2.herokuapp.com/v1');
```

Comment:
I assume not intentional... How about an option `--localhost` that overwrites `--url`, the port for localhost is hardcoded in the configs anyways...